### ISSUE TITLE
Design: Determine how to display Veterans Crisis Line info in footer on screens 767px wide and smaller

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-cms#10348]
As a Veteran,
I want to see the VCL link (which opens the VCL modal) always present in the footer
so that it's easy to find and click.

### Issue report
Currently, Veterans Crisis Line (VCL) call to action is in both the header and the footer on screen sizes 768px and above. In the footer, the heading **In Crisis? Get Help Now**, followed by a link to trigger the VCL modal does not appear on screen sizes 767px and below.  My understanding is that this is intentional, and I would like to propose that it displays across all screen sizes.

**Why?**

- Although the VCL info displays at the top of the page, it's possible a user could miss it. 
- A user could be triggered into crisis mode while filling out a form or reading content on VA.gov and going down the page toward the footer

I discussed this with @jenstrickland (former accessibility specialist) and she also shared a piece of relevant WCAG: Users should not be required to remember data from one page (or view, or step of a process) to another, unless the fundamental purpose of the content requires them to do so. It is highly likely a person will not remember that the VCL was at the top of the page by the time they are elsewhere on the page. By also displaying it in the footer, it is more intuitive. 

This came up as a result of issue department-of-veterans-affairs/va.gov-team#2799, and one suggestion was to have it display in the contact us accordion.  I would recommend having it display above the accordion, as a user in crisis mode may not understand crisis services are available and think to click in there to find VCL info. 

## Acceptance Criteria
- [x] Review available design components to use
- [x] Provide mockup (Sketch file) to include Veterans Crisis Line in the mobile footer, not as accordion

**Comment by jilladams** [comment]
Refined to be a Design ticket to determine where / how in the footer this should appear, to update Sketch files for implementation.

**Comment by fmccaf1** [comment]
I'm thinking we just re-use the red VCL banner that is used on the top nav. 

[Sketch file here](https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857)

Here's a screenshot: 

![VCL banner on bottom nav](https://user-images.githubusercontent.com/106624475/188723977-ba8ad502-38ad-4f46-9b50-ca368b231a5e.jpg)

**Comment by andaleliz** [comment]
Makes sense to me! I would just double check w/ a11y folks that there are no concerns about having repetitive buttons on a page.

**Comment by artsymartha68** [comment]
1. Will this be on all pages. 
2. Have you checked with the VCL owners that this is recommended? In past discussions with them, there is a fine line btwn making the VCL available and promoting it *too* much because folks may use it for unintended reasons.

**Comment by fmccaf1** [comment]
I reached out on the #accessiblility-help channel. [Here's a link to that discussion.](https://dsva.slack.com/archives/C8E985R32/p1662562985208049)

To add specific detail here, Josh Kim stated:

_Assuming focus is managed appropriately and they have unique IDs, this should be fine :smile:
In other words, if I were to open up the veteran crisis line using the button at the bottom of the page, and then close the veteran crisis line modal, I shouldn't be returned to the button at the top of the page (I should be returned back to the button at the bottom of the page).
The main cons would be either (a) potential confusion around multiple, duplicative buttons or (b) crowding a screen reader user's element or form list; but I don't believe either of those outweigh the benefit of providing such a critical feature twice for added visibility._

**Comment by fmccaf1** [comment]
@wesrowe I'm thinking you might have answers to Martha's questions above? Since this ask came from Dave C, I'm assuming VCL would be good with it but would be good to check.

**Comment by artsymartha68** [comment]
Answering both of my questions per side convos with Dave and Danielle:

1. Yes.
2. It's ok!

**Comment by jilladams** [comment]
^ Created an implementation ticket. I think this can now close. Will let Florence do the honors if she agrees.
